frontend/src/crosssection/views/CrossHome.vue

```diff
@@ -15,13 +15,57 @@ Licensed under the Apache License, Version 2.0 (the "License");
   <b-card class="container p-1">
       <h1 class="card-title" id="CrossSectionTitle">Cross Section Home</h1>
       <div>
+        <div>
+          <MapLoadingSpinner :loading="loadingMap"/>
+          <CrossMap
+            :focusedWells="focusedWells"
+          />
+        </div>
       </div>
     </b-card>
 </template>
 
 <script>
+import MapLoadingSpinner from '../../common/components/MapLoadingSpinner.vue'
+import CrossMap from '../components/CrossMap.vue'
+
 export default {
   name: 'CrossHome',
+  components: {
+    CrossMap,
+    MapLoadingSpinner
+  },
+  data () {
+    return {
+      scrolled: false,
+      loadingMap: false,
+      focusedWells: [],
+      mapServerErrorMessage: null,
+      showMapErrorMessage: false,
+    }
+  },
+  methods: {
+    handleScroll () {
+      this.scrolled = window.scrollY > 100
```

Comment:
What does this func do?

Reply:
Removed, it is used on the well search page to monitor when user has scrolled but I am not seeing how it adds function to the page.

Reply:
removed